File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Bahasa C/PRAK601-2310817120009-AlysaArmelia.c
#include <stdio.h>

int main()
{
    int kolom1;
    int kolom2;
    int baris1;
    int baris2;

    scanf("%d %d", &kolom1, &kolom2);

    int matriks [kolom1][kolom2];
    
    for (baris1 = 0; baris1 < kolom1; baris1++)
    {
        for (baris2 = 0; baris2 < kolom2; baris2++)
        {
            scanf("%d", &matriks [baris1][baris2]);   
        }
    }

    for (baris1 = 0; baris1 < kolom1; baris1++)
    {
        for (baris2 = 0; baris2 < kolom2; baris2++)
        {
            printf("%d ", matriks[baris1][baris2]);

        }
    printf("\n");
    }
return 0;
}

File: 2310817120009_AlysaArmelia_PemrogramanI_Modul6/Python/PRAK601-2310817120009-AlysaArmelia.py
kolom1, kolom2 = map(int, input().split())

baris1 = list(map(int, input().split()))

baris2 = 0

while(baris2 < len(baris1)):
    if baris2 % kolom2 == 0 and baris2 > 0:
        print()
    print(baris1 [baris2], end=" ")
    baris2+=1